CustomText.Utilities.CognitiveSearch/Solution/CustomTextSkillsetAzureFunction/Program.cs

```diff
@@ -28,6 +25,9 @@ public static class Program
         {
             log.LogInformation("LUIS-D Extractor and Classifier function");
 
+            var customtextEndpoint = req.Headers["endpointUrl"];
```

Comment:
need some checks for these headers!
we need to make sure the fields are not empty and valid
the azure function should be self-sufficient

should include a new error if these secrets aren't working